app/Meta.py

```diff
@@ -0,0 +1,564 @@
+        shp_file_target = os.path.join(os.getcwd(),UPLOAD_FOLDER,shp_file.filename)
+    
+        #Reading the shapefile as DataFrame
+        data_g=gpd.read_file(shp_file_target)
+    
+        #Reading the csv as Dataframe
+        data = pd.read_csv(fileName,delimiter = ',')
+
+        
+        data['geometry'] = data.geometry.astype(str)
+        
+        geos.WKBWriter.defaults['include_srid'] = True
+
+        
+        for i in range(data_g.shape[0]):
+            p=data_g.iat[i,10]
+            geos.lgeos.GEOSSetSRID(p._geom, 4326)
+            data.iat[i,6]=p.wkb_hex
+
+        #Checking necessary columns are there.
+        columns=data.columns.values.tolist()
+        accepted_columns=['sitename','city','state','country','notes','greenhouse','geometry','time_zone','soil','soilnotes']
+        
+        if(all(x in accepted_columns for x in columns)):
+            
+            data['notes']=data_g['notes']
+
+            data['soil'].fillna("some text", inplace = True)
+            
+            data['soilnotes'].fillna("some text", inplace = True)
+            
+            data['greenhouse'].fillna("f", inplace = True)
+            
+            
+            data=data.fillna('')
+
+            file_name='sites_n.csv'
+
+            data.to_csv(file_name, encoding='utf-8', index=False)
+
+            #Inserting in Bety
+            insert_sites_table(table='sites',data=file_name)
+            msg = {'Message' : 'Successfully inserted',
+                   'Table Affected' : 'Sites',
+                   'Lines Inserted': data_g.shape[0]}
+            
+            return make_response(jsonify(msg), 201)
+
+        else:
+            msg = {'Message' : 'File not acceptable.Check the format of file or columns'}
+            return make_response(jsonify(msg), 400)
+    
+    except OperationalError:
+        # Logs the error appropriately
+        logging.error(traceback.format_exc())
+        return 500
+    except IntegrityError:
+        # Logs the error appropriately
+        logging.error(traceback.format_exc())
+        return 409
+    except Exception as e:
+        # Logs the error appropriately
+        logging.error(traceback.format_exc())
+        return 410        
+
+def insert_treatments(username,fileName):
+    """
+    This function responds to a request for  /yaba/v1/treatments
+    with csv file.Insertion is done to "treatments" table.
+
+    :fileName:      CSV  with treatments meta data
+    :return:        201 on success
+                    400 if file is unsuitable or does not contain appropriate columns
+                    409 Intregrity or Constraint error : 23503 foreign_key_violation | 23505 unique_violation
+                    500 Database Connection Error
+                    401 Unauthorized | No user exists
+                    410 Default error.See logs for more information
+    """
+    try:
+        user_id=fetch_id(username,table='users')
+
+        if user_id =='':
```

Comment:
For this check, `if not user_id:` might be preferred since it'll work with None, empty strings, etc. Maintainers don't need to worry about the fetch_id return type